Sources/AsyncHTTPClient/Utils.swift

```diff
@@ -147,3 +148,37 @@ extension Connection {
         }.recover { _ in }
     }
 }
+
+extension SpanStatus {
+    /// Map status code to canonical code according to OTel spec
+    ///
+    /// - SeeAlso: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#status
+    init(_ responseStatus: HTTPResponseStatus) {
+        switch responseStatus.code {
```

Comment:
I think mapping of HTTP status code (UInt) to `SpanStatus` is very much reusable and as such should be provided in `TracingInstrumentation`, sth like

```swift
SpanStatus(code: UInt, message: String?)
``` 

otherwise each library making HTTP calls and not using AHC will need to map it on its own

Reply:
Hah, this reminded me I had an not submitted review here, yeah this seems like a good candidate to move up into OpenTelemetryInstrumentationSupport/OpenTelemetrySemanticConventions 👍

Reply:
Or not such a good candidate after all 😁👉 https://github.com/slashmo/gsoc-swift-tracing/issues/134